GCLI: base_sha f00b4rc01dc0fee
This is just a global comment.

It should not end up in the patch prelude but in the patch series
prelude.
From 361f83923b9924a3e8796b0ddf03f768e26a1236 Mon Sep 17 00:00:00 2001
From: Nico Sonack <nsonack@herrhotzenplotz.de>
Date: Sat, 16 Sep 2023 22:28:33 +0200
Subject: [PATCH 1/2] Update README.md

---
 README.md | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/README.md b/README.md
index 5ab731f..2b99524 100644
--- a/README.md
+++ b/README.md
@@ -1,3 +1,6 @@
 # README
 
 Das hier ist nur ein kurzer Test.
> Why so much whitespace?
{
+
+
}
+Ich füge zum Test hier mal eine neue Zeile ein.

From d9cbace712a92fdd0bac4f08b6d42e75069af363 Mon Sep 17 00:00:00 2001
From: Nico Sonack <nsonack@herrhotzenplotz.de>
Date: Wed, 20 Sep 2023 20:09:58 +0200
Subject: [PATCH 2/2] Second commit

This is the body of the commit.
---
 README.md | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/README.md b/README.md
index 2b99524..90a258f 100644
--- a/README.md
+++ b/README.md
@@ -4,3 +4,11 @@ Das hier ist nur ein kurzer Test.
 
 
 Ich füge zum Test hier mal eine neue Zeile ein.
> Why all this whitespace?
{
+
+
+
+
+
+
+
}
+This line belongs to a different commit.
